docs/source/platform/schema-validation.md

```diff
@@ -1,60 +1,101 @@
 ---
 title: Validating schema changes
-description: How to validate your schema in your existing CI workflow
+description: How to maintain the schema's contract with CI
 ---
 
-The Apollo GraphQL Platform allows developers to confidently iterate a GraphQL schema by validating the new schema against field-level usage data from the previous schema. By knowing exactly which clients will be broken by a new schema, developers can avoid inadvertently deploying a breaking change.
+To recognize the need for schema validation, first understand that a GraphQL schema defines a contract between clients and server that contains the available types and their behavior. When a GraphQL API is deployed, consumers start to request fields and depend on the contract. If the schema is updated, such as adding a field or removing a type, the contract changes. Modifying the schema and contract can have a wide range of impact on clients from positive, more functionality, to adverse, active schema dependencies no longer exist.
```

Comment:
This first sentence rubs me the wrong way.

Reply:
It might be useful to talk generally about API contracts first and mention that it's OK to modify the contract in GraphQL (unlike other frameworks), but you need to be very conscientious about how you do it.

Reply:
The last sentence would be better structured with parentheticals than appositives (i.e. positive (more functionality) to adverse (active clients are broken in production)), imo